pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -58,6 +60,8 @@ const (
 	// ReasonConditionCheckFailed indicates that the reason for the failure status is that the
 	// condition check associated to the pipeline task evaluated to false
 	ReasonConditionCheckFailed = "ConditionCheckFailed"
+
+	ReasonRunOnStateConflicted = "RunOnStateConflicted"
```

Comment:
a bit confused by this...could you add a comment explaining this state and why/when it can occur?

Reply:
Done :)